releng/config-forker/main.go

```diff
@@ -104,6 +104,13 @@ func generatePresubmits(c config.JobConfig, version string) (map[string][]config
 	return newPresubmits, nil
 }
 
+func shouldDecorate(c *config.JobConfig, util config.UtilityConfig) bool {
+	if util.Decorate != nil {
+		return *util.Decorate
+	}
+	return c.DecorateAllJobs
```

Comment:
why not do the same as https://github.com/kubernetes/test-infra/pull/20656/files?file-filters%5B%5D=.go#diff-88e6b42a331a7aa9260f9a830961e3ba3c77e0e6d781465378b510d7abc1517cR1364?

Reply:
I see, thank you for the context. I think it would be nice to make the generated config more explicit for better visibility, unless it somehow has a problem

Reply:
I agree, but I think that should be done in a separate PR if is is desired. That code is [owned by releng](https://github.com/kubernetes/test-infra/blob/492140b4514ce0b410cb3f0681d0941a67dbc394/OWNERS_ALIASES#L229-L235) so we should get their approval for any behavioral changes, which I don't want to mix into this large PR.